bootstrap/bootstrap.go

```diff
@@ -57,10 +57,33 @@ func Start(ctx context.Context, cfg *config.Config) error {
 		}
 	}
 
+	// create access client with cross-spork capabilities
+	currentSporkClient, err := grpc.NewClient(cfg.AccessNodeHost)
```

Comment:
did you intend to include these changes in this PR?

Reply:
It's needed by follow-up PR, it would be better to have them there but this is really minimal move. I realised after I should change logging first.